module/modmanager/manager.go

```diff
@@ -203,6 +203,21 @@ func (mgr *Manager) Handles() map[string]modlib.HandlerMap {
 	return res
 }
 
+// / this function
+func whiteListViamModules(confs ...config.Module) (
+	whitelisted bool, newConfs []config.Module,
+) {
+	for _, conf := range confs {
+		if strings.Contains(conf.ModuleID, "viam:raspberry-pi") {
```

Comment:
Is checking for a substring in the module ID really "unique" at all? Couldn't I make a module called `viam:raspberry-pievil` that does something terrible to the Try Viam rovers?

Reply:
You could, but we own the viam namespace and upload rights there, so if someone does that, I know who to look for. Should I make it an equal check instead?

Reply:
Gotcha. How would you feel about a const `allowlistedModules map[string]bool` so we can add other requirements for Try Viam in later PRs? I think we'll want to do exact equality checks, yeah.